channels/br.m3u

```diff
@@ -1,4 +1,16 @@
 #EXTM3U x-tvg-url="http://infinitytv.xyz/guide.xml"
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="" group-title="",98 Live (Belo Horizonte) 
+https://cdn.jmvstream.com/w/LVW-8202/ngrp:LVW8202_WXoGmOx1th_all/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="" group-title="",Agro Brasil TV
+http://45.162.230.234:1935/agrobrasiltv/agrobrasiltv/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/vpzYZDT.png" group-title="",ALL Sports
+https://5cf4a2c2512a2.streamlock.net/dgrau/dgrau/chunklist.m3u8
+#EXTINF:-1 tvg-id="band.br" tvg-name="Band" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/h8drbLl.png" group-title="",Band
+http://evpp.mm.uol.com.br:1935/geob_band/app/playlist.m3u8
+#EXTINF:-1 tvg-id="bandnews.br" tvg-name="Band News" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/W5xuts0.png" group-title="",Band News
+https://evpp.mm.uol.com.br/geob_band/bandnewstv/chunklist_w5392006.m3u8
+#EXTINF:-1 tvg-id="bandsports.br" tvg-name="Band Sports" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/TG25xuW.png" group-title="",Band Sports
+https://evpp.mm.uol.com.br/geob_band/bandsports/playlist.m3u8
```

Comment:
Links does not work. 
Tested in Elmedia Video Player, VLC Player and streamtest.in

Reply:
Seems that these links are geo-restricted to Brazil.